jck/jtrunner/JavaTestRunner.java

```diff
@@ -536,13 +536,15 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 					if ( !platform.equals("win") ) {
```

Comment:
Btw, out of curiosity, what happens if you put the check here, since win and osx sort of have the same behaviour once mac machines configured for auto-logon: 
`if ( !platform.equals("win") || !platform.equals("osx") ) {`

Reply:
Yes, tried it. Unlike Windows, OSX needs the two configuration options to be set : 

```
fileContent += "set jck.env.testPlatform.headless No" + ";\n";
fileContent += "set jck.env.testPlatform.xWindows Yes" + ";\n";
```

Ref : Grinder/21052.